test/kitchen/drivers/ec2-driver.yml

```
@@ -113,6 +113,12 @@ platforms:
     <% if ENV["KITCHEN_CWS_PLATFORM"] %>
     cws_platform: <%= ENV["KITCHEN_CWS_PLATFORM"] %>
     <% end %>
+    <% if ENV["DOCKER_REGISTRY_LOGIN"] && ENV["DOCKER_REGISTRY_PASSWORD"] %>
+    docker:
+      registry: <%= ENV["DOCKER_REGISTRY_URL"] %>
+      username: <%= ENV["DOCKER_REGISTRY_LOGIN"] %>
+      password: <%%= ENV["DOCKER_REGISTRY_PASSWORD"] %>
```

```suggestion
      password: <%= ENV["DOCKER_REGISTRY_PASSWORD"] %>
```
I don't know if the double `%` was on purpose because it's a password or if it's a typo

---

It was intentional. This way the password is not in the generated `kitchen.yml` file.